admin/settings.yaml

```diff
@@ -282,6 +282,16 @@ c-toxcore:
           - "restyled"
           # Custom
           - "CodeFactor"
+          - "analysis (autotools)"
```

Comment:
How does the namespacing on these work? Dropped on GitHub Actions-checks, included on reports from other services?

Reply:
I'm not sure what you mean, but namespaces happen when you use imported rules like the "common" ones.

Reply:
Parentheses come from the matrix strategy.